test/escapeMarkdown.test.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-/* eslint-disable max-len, no-undef */
+/* eslint-env jest */
+/* eslint-disable max-len */
```

Comment:
I noticed that `jest` is listed as a dev dependency in `package.json`, so I enabled the `jest` ESLint environment so that `no-undef` is not needed.